lib/spack/spack/binary_distribution.py

```diff
@@ -739,7 +739,7 @@ def get_buildfile_manifest(spec):
     # Non-symlinks.
     for rel_path in visitor.files:
         abs_path = os.path.join(root, rel_path)
-        m_type, m_subtype = fsys.mime_type(abs_path)
+        m_type, m_subtype = fsys.binary_mime_type(abs_path)
```

Comment:
@johnwparent this code also wants to identify text file types (IIRC we discussed this but apparently I never left this as a comment, so apologies for that)

Reply:
Specifically, see a few lines below where this calls `relocate.needs_text_relocation`